### PR TITLE
Fix duping of shotgun hull for scrap loaded shot

### PR DESCRIPTION
**Pull request by RenechCDDA**
#### Summary

Bugfixes "Return only one shotgun hull per shot disassembled"

#### Purpose of change

Ammo recipes that result in multiple rounds will disassemble into that many number of casings *per-round* disassembled.  As this produces casings out of thin air ("duping"), it should be fixed

#### Describe the solution

Changing all ammo recipes to produce 1 ammo is a band-aid fix. All ingredients and time required were also halved, but rounded up. This results in a slight nerf to those components.

#### Describe alternatives you've considered

Rounding down components - disregarded, but no objections to it. Just wanted to be conservative
Resolving the fundamental issue - This requires C++ knowledge I do not have

#### Testing

Debug character spawned in the necessary items to make 10, confirmed duplication. Save, quit, made change, reloaded, started with a fresh set and confirmed I could not create more no matter how many craft--->disassemble cycles I took.

#### Additional context

The scrap shot recipe seems to have a carefully measured formulation already at it with a comment noting that it produces 2. I believe this is important to retain so it can be returned to later, so I added another comment indicating that this band-aid fix has been applied.

I also checked the other ammo recipes in the JSON to see if any others were susceptible to this duplication. The scrap-loaded shot recipes were the only ones I found.

### CODE REVIEW

File: data/json/recipes/ammo/shot.json [hunk 1]
```diff
@@ -263,28 +263,29 @@
...
     "batch_time_factors": [ 50, 5 ],
     "autolearn": true,
-    "charges": 2,
+    "charges": 1,
     "reversible": true,
-    "using": [ [ "ammo_shot", 2 ] ],
+    "using": [ [ "ammo_shot", 1 ] ],
     "tools": [ [ [ "press", -1 ] ] ],
     "//": "1166 mg gunpowder rounded to 12 100 mg 'pieces'; recipe makes two shells",
+	"//1": "currently halved inputs and result pending fix of casing duplication",
```
Comment (actual-nh) [inline data/json/recipes/ammo/shot.json:274]:
```suggestion
	  "//1": "currently halved inputs (rounding up) and result pending fix of casing duplication",
```

File: data/json/recipes/ammo/shot.json [hunk 2]
```diff
@@ -300,24 +301,24 @@
     "time": "2 m",
     "batch_time_factors": [ 50, 5 ],
     "autolearn": true,
-    "charges": 2,
+    "charges": 1,
     "reversible": true,
-    "using": [ [ "ammo_shot", 2 ] ],
+    "using": [ [ "ammo_shot", 1 ] ],
     "tools": [ [ [ "press", -1 ] ] ],
     "components": [
```
Comment (actual-nh) [inline data/json/recipes/ammo/shot.json:308]:
```suggestion
    "//1": "currently halved inputs (rounding up) and result pending fix of casing duplication",
    "components": [
```

File: data/json/recipes/ammo/shot.json [hunk 3]
```diff
@@ -328,23 +329,23 @@
     "activity_level": "LIGHT_EXERCISE",
     "copy-from": "bp_shot_scrap",
     "book_learn": [ [ "survival_book", 1 ], [ "mag_survival", 1 ] ],
-    "time": "4 m",
+    "time": "2 m",
     "batch_time_factors": [ 50, 5 ],
     "tools": [ [ [ "press_dowel", -1 ] ] ],
     "components": [
```
Comment (actual-nh) [inline data/json/recipes/ammo/shot.json:335]:
```suggestion
    "//1": "currently halved inputs (rounding up) and result pending fix of casing duplication",
    "components": [
```

**Review by actual-nh — COMMENTED** [review]
Recipe charges, etc are a rather confusing area...

File: data/json/recipes/ammo/shot.json [hunk 4]
```diff
@@ -263,28 +263,29 @@
...
     "batch_time_factors": [ 50, 5 ],
     "autolearn": true,
-    "charges": 2,
+    "charges": 1,
     "reversible": true,
-    "using": [ [ "ammo_shot", 2 ] ],
+    "using": [ [ "ammo_shot", 1 ] ],
     "tools": [ [ [ "press", -1 ] ] ],
     "//": "1166 mg gunpowder rounded to 12 100 mg 'pieces'; recipe makes two shells",
+	  "//1": "currently halved inputs (rounding up) and result pending fix of casing duplication",
```
Comment (BrettDong) [inline data/json/recipes/ammo/shot.json:274]: Use space characters for indentation:
```suggestion
    "//1": "currently halved inputs (rounding up) and result pending fix of casing duplication",
```

**Comment by actual-nh** [comment]
@Maleclypse: Could you trigger the JSON formatting workflow? Thanks!

**Pull request merged**